shared/build.gradle.kts

```diff
@@ -37,27 +39,56 @@ kotlin {
                 implementation(compose.material)
                 @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
                 implementation(compose.components.resources)
+                api("dev.icerock.moko:mvvm-core:0.15.0")
+                api("io.ktor:ktor-client-core:2.2.4")
+                api("dev.icerock.moko:network:0.20.1")
+                api("dev.icerock.moko:resources:0.20.1")
+//                api("dev.icerock.moko:resources-compose:0.20.1")
+
+                //Could not resolve all dependencies for configuration ':shared:iosSimulatorArm64CompileKlibraries'.
+                //Could not resolve dev.icerock.moko:biometry:0.2.0.
+                //implementation("dev.icerock.moko:biometry:0.2.0")
             }
         }
+
         val androidMain by getting {
             dependencies {
                 api("androidx.activity:activity-compose:1.6.1")
                 api("androidx.appcompat:appcompat:1.6.1")
                 api("androidx.core:core-ktx:1.9.0")
+                api("io.ktor:ktor-client-okhttp:2.2.4")
+                api("dev.icerock.moko:biometry:0.2.0")
```

Comment:
it's not androidOnly lib

Reply:
cannot resolve these libraries in common code